sor/src/main/java/com/bazaarvoice/emodb/sor/db/DeltaIterator.java

```diff
@@ -56,6 +56,7 @@ private ByteBuffer reverseCompute() {
 
         // delta is fragmented if the first block is not zero. In this case, we skip it.
         if (getBlock(_list.get(0)) != 0) {
+            _list.clear();
```

Comment:
I think we could add a detail comment here about why clearing the list is needed, and how the buffer overflow exception can creep in if we don't do this.

Reply:
👍